src/DB/usersDB.mjs

```diff
@@ -38,13 +38,17 @@ export default async function (db) {
       applogger.trace('Searching for user "' + user._key)
       return user
     },
+    // NEW GET ROLE TYPES FUNCTION
     async getRoleTypes () {
       let query = 'FOR user IN users RETURN DISTINCT user.role'
```

Comment:
this was not needed! we only have 3 types of users and they are known beforehand: 'admin', 'participant' and 'researcher'.
You can probably delete this

Reply:
Alright! So we delete it from the API, as well as remove the role type filter from the web interface?

Reply:
the filter is useful on the interface, but the array can be hard coded instead of being built from asking the server